docs/refman/CMakeLists.txt

```diff
@@ -17,6 +17,12 @@ if (ENABLE_REFMAN)
   # have to specify it because we have another directory to strip.
   set(DOXYGEN_STRIP_FROM_PATH "${PROJECT_SOURCE_DIR}/include/openenclave/;${PROJECT_SOURCE_DIR}/docs/refman")
 
+  # Allow @experimental to be used in headers to mark an API as experimental and add it to a list of
+  # experimental APIs.
+  set(DOXYGEN_ALIASES
+    "experimental = \\xrefitem experimental \\\"This feature is marked as experimental\\\" \\\"Experimental List\\\""
+    "experimental{1} = \\xrefitem experimental \\\"Experimental\\\" \\\"Experimental List\\\" \\1")
```

Comment:
What is the `experimental{1}` alias for? What would someone pass as a parameter to the tag? Maybe an example used in a comment would help understanding.

Reply:
Together, these cause a separate "experimental.html" page to be created that has a list of all experimental APIs.  Tagging an API with @experimental then causes the notice to show up on that API's own page, and the notice is hotlinked to the experimental.html page.   

The {1} is not for literal use, it means there's one parameter in the expansion (the \1 gets expanded).   I'm not the expert, but I tested it and verified it works, and this snippet appears in other open source projects and stackoverflow threads.

See http://www.doxygen.nl/manual/custcmd.html#custcmd_complex and http://www.doxygen.nl/manual/commands.html#cmdxrefitem for more info.

Reply:
Good enough for me :)

Reply:
Ok, here's more details from testing:

@experimental
causes the following to show up (linked to the Experimental List page):
        This feature is marked as experimental:

@experimental FooBar
casues the following to show up (linked to the Experimental List page):
        This feature is marked as experimental:
                FooBar

So you use no argument if the entire API is experimental.
You use an argument if some particular subset or variation is experimental.